index-de.html

```diff
@@ -29,89 +29,57 @@ <h1 id="top">Ein grafischer Spickzettel für Git</h1>
-    Vektorgrafiken nicht angezeigt werden, kann die
-    <a
-    href="?no-svg">SVG freie</a>
-    Variante gewählt werden.
-  </p>
+  <p id="link-to-png">Sollten die Grafiken nicht funktionieren, kannst du die
+    <a href="?no-svg">SVG-freie</a> Version dieser Seite auswählen.</p>
 
-  <p id="link-to-svg">
-    SVG Vektorgrafiken sind deaktiviert.
-    <a href="index.html">(SVG aktivieren)</a>
-  </p>
+  <p id="link-to-svg">Die Grafiken sind derzeit nicht im SVG-Format.
+    <a href="index-de.html">(zur SVG-Version)</a></p>
 
-  <p>
-    Diese Seite ist eine Sammlung elementarer git Befehle. Grundwissen sollte sich hier vertiefen lassen. Wie diese Seite erstellt wurde wird im
-    <a href='http://github.com/MarkLodato/visual-git-guide'>
-      GitHub
-    repository
-    </a>
-    erklärt.
-  </p>
+  <p>Diese Seite ist eine kurze grafische Referenz der am häufigsten verwendeten
+    Git-Befehle. Wenn du schon einige Git-Grundlagen kennst, kannst du mit
+    dieser Seite dein Verständnis vertiefen. Wenn du an der Erstellung dieser
+    Seite interessiert bist, sieh dir mein
+    <a href='http://github.com/MarkLodato/visual-git-guide'>GitHub-Repository</a>
+    dazu an.</p>
+
+  <p id="link-to-d3">Ebenfalls empfohlen: <a
+    href="http://onlywei.github.io/explain-git-with-d3/#">Visualizing Git
+    Concepts with D3</a> (englisch).</p>
 
   <h2 id="contents">Inhalt</h2>
   <ol>
-    <li>
-      <a href="#basic-usage">Grundlegende Benutzung</a>
-    </li>
-    <li>
-      <a href="#conventions">Konventionen</a>
-    </li>
-    <li>
-      <a href="#commands-in-detail">Kommandos im Detail</a>
+    <li><a href="#basic-usage">Grundlagen</a></li>
+    <li><a href="#conventions">Konventionen</a></li>
+    <li><a href="#commands-in-detail">Kommandos im Detail</a>
       <ol>
-        <li>
-          <a href="#diff">Diff</a>
-        </li>
-        <li>
-          <a href="#commit">Commit</a>
-        </li>
-        <li>
-          <a href="#checkout">Checkout</a>
-        </li>
-        <li>
-          <a href="#detached">Commit mit losgelöstem (detached) HEAD</a>
-        </li>
-        <li>
-          <a href="#reset">Reset</a>
-        </li>
-        <li>
-          <a href="#merge">Merge</a>
-        </li>
-        <li>
-          <a href="#cherry-pick">Cherry Pick</a>
-        </li>
-        <li>
-          <a href="#rebase">Rebase</a>
-        </li>
+        <li><a href="#diff">Diff (Unterschied)</a></li>
+        <li><a href="#commit">Commit (Festlegung)</a></li>
+        <li><a href="#checkout">Checkout (Ausprobieren)</a></li>
+        <li><a href="#detached">Commit mit detached (losgelöstem) HEAD</a></li>
+        <li><a href="#reset">Reset (Zurücksetzen)</a></li>
+        <li><a href="#merge">Merge (Mischen)</a></li>
+        <li><a href="#cherry-pick">Cherry Pick (Kirschen bzw. Rosinen rauspicken)</a></li>
+        <li><a href="#rebase">Rebase (Basis wechseln)</a></li>
```

Comment:
Bei "$ man git commit" steht "git-commit - Record changes to the repository", ```(Übergeben)``` könnte also besser passen.
Wenn der Befehl ausgeführt werden die Veränderungen, die mit add markiert wurden, in ein commit gefasst und dem repository übergeben (oder so ähnlich).